packages/gamut/src/GridForm/__tests__/GridForm-test.tsx

```diff
@@ -19,44 +17,63 @@ import {
   stubTextField,
 } from './stubs';
 
+const fields = [stubCheckboxField, stubSelectField, stubTextField];
+
+const renderView = setupRtl(GridForm, {
+  fields,
+  showRequired: false,
+  submit: { type: 'fill', contents: <>Submit</>, size: 6 },
+});
+
+type renderViewReturn = ReturnType<typeof renderView>;
```

Comment:
Nit: this is generally done:

```suggestion
type RenderViewReturn = ReturnType<typeof renderView>;
```